freedomofpress/redmine_openpgp
# Allow disabling the delete button on the Server Key form (rely on rake task)

As a precaution, adds a very simple and superficial checkbox setting that prevents the 'Remove' button appearing at /pgp for the 'Redmine Server' key - this helps admins avoid accidentally deleting the server key instead of their own key (as the form is a bit confusing!).

The assumption is that you will use other means to insert/update the server key (e.g the rake task, via something like Ansible), which are not affected by this change.

To test:

Visit `/settings/plugin/openpgp` and toggle on/off the checkbox setting 'Disable ability to update server key in UI (use Rake only)'. Each time, compare at `/pgp` to make sure the 'Remove' button disappears/reappears accordingly at the bottom of the form (for the 'Redmine Server' key).
